l10n_es_aeat_mod347/__manifest__.py

```diff
@@ -10,7 +10,7 @@
 
 {
     'name': "AEAT modelo 347",
-    'version': "11.0.1.2.0",
+    'version': "11.0.1.2.1",
```

Comment:
Sólo como información, ahora no hace falta cambiar el número de versión. Lo puede hacer el bot al fusionar: https://odoo-community.org/blog/the-oca-blog-1/post/the-oca-github-bot-can-now-merge-pull-requests-92, pero bueno, lo puedes dejar ya si quieres.